### PR TITLE
LOG-6206: trim newline symbol for username/password

### PR DESCRIPTION
**Pull request by vparfonov**
### Description

/cc 
/assign 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA: https://issues.redhat.com/browse/LOG-6206
- Enhancement proposal:

### CODE REVIEW

File: internal/generator/vector/output/elasticsearch/elasticsearch.go [hunk 1]
```diff
@@ -238,8 +238,8 @@ func BasicAuth(id string, o logging.OutputSpec, secret *corev1.Secret) []Element
 		if common.HasUsernamePassword(secret) {
 			hasBasicAuth = true
 			up := UserNamePass{
-				Username: common.GetFromSecret(secret, constants.ClientUsername),
-				Password: common.GetFromSecret(secret, constants.ClientPassword),
+				Username: strings.TrimRight(common.GetFromSecret(secret, constants.ClientUsername), "\n"),
```
Comment (jcantrill) [inline internal/generator/vector/output/elasticsearch/elasticsearch.go:241]: I would recommend to trim both ends

File: internal/generator/vector/output/elasticsearch/elasticsearch_test.go [hunk 2]
```diff
@@ -132,1573 +130,1573 @@ user = "testuser"
...
-bulk.action = "create"
-encoding.except_fields = ["write_index"]
-id_key = "_id"
-api_version = "v9"
-
-[sinks.es_1.request]
-timeout_secs = 2147483648
-`,
-		}),
+		//		Entry("with tls key,cert,ca-bundle", helpers.ConfGenerateTest{
```
Comment (jcantrill) [inline internal/generator/vector/output/elasticsearch/elasticsearch_test.go:133]: Why?

Comment (vparfonov) [reply internal/generator/vector/output/elasticsearch/elasticsearch_test.go:133]: ooops, by mistake

**Review by jcantrill — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**